OnboardIos/OnboardIos/App/AppDelegate.m

```diff
@@ -21,9 +21,21 @@ @implementation AppDelegate
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
     // Override point for customization after application launch.
     self.window = [[UIWindow alloc] initWithFrame:[[UIScreen mainScreen] bounds]];
-    [self.window setRootViewController:[ViewController new]];
-    [self.window makeKeyAndVisible];
+    //[self.window setRootViewController:[ViewController new]];
+    //[self.window makeKeyAndVisible];
```

Comment:
unnecessary comment code, if you are not using this code, they don't need to be committed